core/src/color.rs

```diff
@@ -80,16 +115,109 @@ impl Color {
             self.a,
         ]
     }
+
+    #[cfg(feature = "palette")]
+    /// Convert from palette's [`Srgba`] type to a [`Color`]
+    ///
+    /// [`Srgba`]: ../palette/rgb/type.Srgba.html
+    /// [`Color`]: struct.Color.html
+    pub fn from_srgba(srgba: Srgba) -> Color {
+        Color::new(srgba.red, srgba.green, srgba.blue, srgba.alpha)
+    }
+
+    #[cfg(feature = "palette")]
+    /// Convert from [`Color`] to palette's [`Srgba`] type
+    ///
+    /// [`Color`]: struct.Color.html
+    /// [`Srgba`]: ../palette/rgb/type.Srgba.html
+    pub fn into_srgba(self) -> Srgba {
+        Srgba::new(self.r, self.g, self.b, self.a)
+    }
```

Comment:
I think we can remove these two. The `From` traits should be enough.

Reply:
Fixed in d464d1899c9032e4e1b502ff6b9c3a86f41976c7